packages/scss/src/components/progress/vars.scss

```diff
@@ -1,7 +1,8 @@
 @mixin vars {
 	--components-progress-margin-vertical: var(--spacings-S);
 	--components-progress-margin-horizontal: 0;
-	--components-progress-height: var(--spacings-XXS);
+	--components-progress-height: 0.25rem;
```

Comment:
Why no longer use the spacing CSS vars?

Reply:
The component height is not related to spacings (which apply to paddings, margins & gaps).
Basically we don't want to have impact on component height if we edit spacing tokens.